hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/writebehind/MapStoreWriteBehindTest.java

```diff
@@ -576,6 +576,36 @@ public void testDelete_thenPutIfAbsent_withWriteBehindEnabled() {
         assertNull(putIfAbsent);
     }
 
+    @Test
+    public void get_after_clear_loads_data_from_map_loader() {
+        EventBasedMapStore testMapStore = new EventBasedMapStore();
+        Config config = newConfig(testMapStore, 100);
+
+        HazelcastInstance instance = createHazelcastInstance(config);
+        IMap<Integer, Integer> map = instance.getMap("default");
+
+        final int keyCount = 1_000;
+
+        for (int i = 0; i < keyCount; i++) {
+            map.set(i, i);
+        }
+
+        map.clear();
+
+        for (int i = 0; i < keyCount; i++) {
+            map.get(i);
```

Comment:
you could add an assertion that values obtained from `get` are `null` here

Reply:
i think null is not required, any value which is loaded by map-loader is ok.